src/main/java/com/chopshop166/chopshoplib/sensors/MockPIDSource.java

```diff
@@ -1,40 +1,27 @@
 package com.chopshop166.chopshoplib.sensors;
 
-import edu.wpi.first.wpilibj.PIDSource;
-import edu.wpi.first.wpilibj.PIDSourceType;
-import edu.wpi.first.wpilibj.SendableBase;
+import edu.wpi.first.wpilibj.Sendable;
+import edu.wpi.first.wpilibj.interfaces.Potentiometer;
 import edu.wpi.first.wpilibj.smartdashboard.SendableBuilder;
 
 /**
- * A {@link PIDSource} that can be controlled via the dashboard.
+ * A {@link Potentiometer} that can be controlled via the dashboard.
```

Comment:
Not sure what happened here.. Was this supposed to be in `src/main/java/com/chopshop166/chopshoplib/sensors/MockPotentiometer.java`?

Reply:
`PIDSource` was deprecated for this year. Because of this, we can use any `DoubleSupplier` in a place where we would formerly use that class. The sole purpose of the `Potentiometer` interface is to add a `get()` method to `PIDSource`, so I'd argue that we don't actually need that interface at all, and it would be better off not implementing it.

This means that the class can be changed to a generic `Double` control. I'll fix the comment, but it might be good to move it into a separate `.dashboard` package down the line. For now, I'll change it to be a `MockAnalogInput`, since that's probably closest to the actual uses we'd have for it.